src/coreclr/ToolBox/superpmi/mcs/verbjitflags.h

```diff
@@ -0,0 +1,18 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+//----------------------------------------------------------
+// verbJitFlags.h - verb that prints sumary of jit flags values
+//----------------------------------------------------------
+#ifndef _verbJitFlags
```

Comment:
do we have a preference between `#pragma one` and `ifndef` for new files?

Reply:
I think ifndef is preferable; it doesn't depend on support for `pragma once` in compiler.